custom/enikshay/private_sector_datamigration/management/commands/update_user_contact_phone_number.py

```diff
@@ -43,3 +45,7 @@ def handle(self, domain, location_ids, **options):
             if not contact_phone_number and len(user_detail.mobileNumber) == 10:
                 location_user.user_data['contact_phone_number'] = '91' + user_detail.mobileNumber
                 location_user.save()
+                print 'Updated %s' % location_user.user_id
```

Comment:
printing within the loop might mess with the progress bar output, not sure if that's important or not